### ISSUE TITLE
Setup recurring interpolation build

### ISSUE THREAD

**Issue by dianashk** [pelias/pelias#503]
- [x] writeup proposal for needed infrastructure improvements and the end-goal of this task
- [x] research how to best implement what is needed
- [x] implement it
- [ ] high-five everyone!

Connects pelias/pelias#468
Connects pelias/interpolation#7

**Comment by dianashk** [comment]
We've decided to implement this recurring build setup using our current Chef framework and avoid introducing new technologies at this time. We will revisit all of our infrastructure in Q2-17 and make decisions about new technology then.

**Comment by dianashk** [comment]
What's left here before we can high-five everyone? 👏

**Comment by orangejulius** [comment]
I currently have a shell script that sets up an interpolation build from scratch. There's a build currently running and it looks great. However, it will take several days to run still before we know for sure. @missinglink and I have been discussing how to speed up the build and we have some ideas. That might help us get it out faster.

**Comment by orangejulius** [comment]
**UPDATE!** I have been working on these automated builds for a while sporadically when i have time. The script for automated builds is, I think, almost done, but making progress has been slow because the builds take a very long time. The build that is currently running, for example, started on June 6th. So it's been running now for almost 10 days.

**Comment by orangejulius** [comment]
Duplicate of https://github.com/pelias/pelias/issues/468
